Bookish.DataAccess/BookishService.cs

```diff
@@ -13,6 +13,7 @@ public interface IBookishService
         IEnumerable<BookCopy> GetCopies(string filter);
         IEnumerable<BookLoan> GetLoans(string userId);
         void AddBook(string title, string author, string isbn, int totalCopies);
+        bool GetIsbnExists(string isbn);
```

Comment:
For a boolean method like this I would expect it to be called something like a question, e.g. `DoesIsbnExist` 🙂